idea-plugin/src/main/kotlin/org/jetbrains/bunches/ideaPlugin/SwitchAction.kt

```diff
@@ -0,0 +1,49 @@
+package org.jetbrains.bunches.ideaPlugin
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import org.jetbrains.bunches.cleanup.cleanup
+import org.jetbrains.bunches.restore.RESTORE_CLEANUP_COMMIT_TITLE
+import org.jetbrains.bunches.restore.Settings
+import org.jetbrains.bunches.restore.doStepByStepSwitch
+import org.jetbrains.bunches.restore.doSwitch
+
+class SwitchAction : AnAction() {
+    override fun actionPerformed(e: AnActionEvent) {
+        val project = e.project ?: return
+        val suffixes = BunchFileUtils.bunchExtension(project)
```

Comment:
Here is now a bug previously covered by nullable type of `bunchExtension` function.